### ISSUE TITLE
Unable to pass 'CORS_ORIGIN_WHITELIST' environment variable to Soul CLI for CORS configuration

### ISSUE THREAD

**Issue by AbegaM** [thevahidal/soul#82]
### Describe the bug

* When running Soul in development mode, I can create a .env file in the /soul/cors folder and pass environment variables such as PORT, CORS_ORIGIN_WHITELIST, DB, EXTENSIONS, etc. However, when running the Soul CLI, I can only pass the PORT, DB, and RATE_LIMIT values. This is causing CORS issues when using the Soul CLI from other applications.

### To Reproduce

1. install Soul globally.
2. Run Soul from the CLI with a sample database:
    ```
     soul -d .sample.db -p 4000 -r true
    ```
3. Send a request to Soul from another application (e.g., a ReactJS app):
    ```
       http://localhost:4000/api/tables
    ```
4. The application will throw a CORS error.

### Expected behavior
* I expect my ReactJS app not to throw an error, just like it doesn't throw an error when running Soul in development mode and using the .env file to configure the CORS_ORIGIN_WHITELIST environment variable.

### Screenshots
If applicable, please add screenshots to help explain the problem.

### Desktop (please complete the following information):

OS: Mac OS
Browser: Chrome
Version: [insert version here]

**Comment by thevahidal** [comment]
@AbegaM 
Hey Abenezer,
Have you tried setting the `CORS_ORIGIN_WHITELIST` environment variable, e.g.  `export CORS_ORIGIN_WHITELIST=http://localhost:3000` (where `3000` is the port that your React project is using) before running Soul?

**Comment by AbegaM** [comment]
Hello @thevahidal,

Thank you for your response. I have attempted to pass the CORS_ORIGIN_WHITELIST using a .env file, but unfortunately, I have not had any success. When I run Soul in development mode, everything works fine, but my ReactJS app continues to throw a CORS error. I have noticed that you have included all the necessary config variables in the /soul/core/.env.sample file. While I am able to utilize these variables, I am still unable to pass the CORS config when running Soul using the CLI.
